src/cloud/architecture/scaled-architecture.md

```diff
@@ -0,0 +1,102 @@
+
+Vertical scaling must use the same instance type and size for each node:
+
+-  `C5 instances` for each WEB node
+-  `M5 or M5n instances` for each Core node
+
+### Core tier scaling
+
+There are 3 nodes (core nodes) for the database and included services, such as ElasticSearch, MariaDB, and Redis. When the core tier approaches capacity, the only way to scale is to increase the server size, such as boosting the CPU power and memory. Capacity is limited to the size of the node that is available. Because the database cluster is designed for high availability, you cannot scale horizontally in a reliable way with the technologies used.
+
+![Core tier scaling]
+
+Consider an example that the core node instance type is _m5.2xlarge_ with 32Gb RAM. A service, such as the database, uses a considerable amount of memory (30Gb). Scaling to the next available instance size _m5.4xlarge_ provides 64Gb RAM, double the memory, and accommodates the growing needs of the database.
+
+Further optimizing performance includes routing traffic. You can route traffic based on the node type. For example, you can serve web traffic on the db node or you can isolate the db node from the web traffic.
+
+### Web tier scaling
+
+There are 3 nodes for php-fpm and manages web traffic. In addition to vertical scaling by increasing power and memory, the web tier can scale horizontally by adding extra web servers to an existing cluster when constricted at the PHP level.
+
+![Web tier scaling]
+
+This complements the vertical scaling provided by the database tier.
+
+Consider an example that the web node instance type is _C5.2xlarge with 8 CPU and 16Gb RAM_. The number of requests to the site increased greatly. You can add an additional C5.2xlarge node to handle the increase in php-fpm processes or you can change each instance type to _C5.4xlarge with 16 CPU and 32Gb RAM_. Adding an additional node is less expensive than changing all web node instances to a larger type.
+
+## Project structure
+
+Minimally, Pro projects with the Scaling architecture have 6 nodes available.
+
+-  3 web nodes c5.2xlarge (8 CPU, 16 Gb RAM)
+-  3 core nodes m5.2xlarge (8 CPU, 32 Gb RAM)
+
+Each project is unique, however, and requires performance monitoring to properly analyze resource management. Each account includes the New Relic Infrastructure service, which automatically connects with the application data and performance analytics to provide dynamic server monitoring. See [New Relic services][nri].
+
+### SSH access
```

Comment:
`4 to 6` is just in default setup. If it was scaled, it will be `4 to n`

Reply:
Of course! Fixed.